app/views/site/about.erb

```diff
@@ -53,6 +53,8 @@
         once you think about the finer details.</p>
 
         <p>Following that idea, the icon came into existence after Katrina mentioned the idea of <i>the devil is in the details</i> to a designer.</p>
+
+        <p>To learn more about what it means for <i>the devil is in the details</i>, check out these talks by Katrina: <a href="https://www.youtube.com/watch?v=GWEEPt8VvmU">Overkill</a>, <a href=https://www.youtube.com/watch?v=59YClXmkCVM>Succession</a>
```

Comment:
I think this needs to be something like _what it means for the devil to be in the details_ or _what *the devil is in the details* means_.